From 9a1ba049dbe6d30690b09a1f522aa1b5bf15f1e6 Mon Sep 17 00:00:00 2001
From: Josef Schlehofer <pepe.schlehofer@gmail.com>
Date: Sat, 24 Apr 2021 10:54:25 +0200
Subject: [PATCH] foris: update to version 101.1.2

In this release 101.1.2, there was moved ubus socket according to
changes done in OpenWrt
---
 web/foris/foris/Makefile | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/web/foris/foris/Makefile b/web/foris/foris/Makefile
index 39f3cd77e..3c307c76c 100644
--- a/web/foris/foris/Makefile
+++ b/web/foris/foris/Makefile
@@ -8,12 +8,12 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=foris
-PKG_VERSION:=101.1.1
-PKG_RELEASE:=5
+PKG_VERSION:=101.1.2
+PKG_RELEASE:=1
 
 PKG_SOURCE_PROTO:=git
 PKG_SOURCE_URL:=https://gitlab.nic.cz/turris/foris/foris.git
-PKG_MIRROR_HASH:=d9e8ba96063a32f59eede8daaf46a9dd70c263a6dd0b098356a940696d0b7ef4
+PKG_MIRROR_HASH:=1332de0a98a12fd0464c06d65eea4f7f14ea30d375e24f6833b8a0b770147dd2
 PKG_SOURCE_VERSION:=package-v$(PKG_VERSION)
 
 PKG_MAINTAINER:=CZ.NIC <packaging@turris.cz>
-- 
2.25.1

